src/schema/__tests__/fair.test.js

```diff
@@ -461,6 +461,90 @@ describe("Fair", () => {
     })
   })
 
+  describe("formattedDateStatus", () => {
+    const query = gql`
+      {
+        fair(id: "aqua-art-miami-2018") {
+          formattedDateStatus
+        }
+      }
+    `
+
+    const realNow = Date.now
+    beforeEach(() => {
+      Date.now = () => new Date("2019-01-30T03:24:00")
+    })
+    afterEach(() => {
+      Date.now = realNow
+    })
+
+    describe("with future dates", () => {
+      it("The fair is opening", async () => {
+        const fairData = {
+          start_at: "2019–02–06T12:00:56+00:00",
+          end_at: "2019–02–30T12:34:56+00:00",
+        }
+
+        const mockFairLoader = jest.fn(() => Promise.resolve(fairData))
+        context = {
+          fairLoader: mockFairLoader,
+        }
+
+        const data = await runQuery(query, context)
+
+        expect(data).toEqual({
+          fair: {
+            formattedDateStatus: "Opens Feb 6 at 12pm",
+          },
+        })
+      })
+    })
+
+    describe("with running dates", () => {
+      it("The fair is closing", async () => {
+        const fairData = {
+          start_at: "2019–01–20T12:34:56+00:00",
+          end_at: "2019–02–04T12:00:56+00:00",
+        }
+
+        const mockFairLoader = jest.fn(() => Promise.resolve(fairData))
+        context = {
+          fairLoader: mockFairLoader,
+        }
+
+        const data = await runQuery(query, context)
+
+        expect(data).toEqual({
+          fair: {
+            formattedDateStatus: "Closes Feb 4 at 12pm",
```

Comment:
So if we agree to my other comment this should assert that we show minutes as well. 

(Unless it is at the top of the hour, which I suspect is the _only_ case in which we may reasonably drop the `:00` — could be good to run this by design to confirm the various scenarios)

Reply:
Yes good point I'll add the minutes when applicable.

Reply:
Just updated the function and added a test to check minutes